main.go

```diff
@@ -75,7 +75,7 @@ var (
 	hostname       = iniConf.String("hostname", "unknown-hostname", "Override the hostname we get from the OS")
 	httpProxy      = iniConf.String("httpProxy", "", "Proxy server for HTTP requests")
 	httpsProxy     = iniConf.String("httpsProxy", "", "Proxy server for HTTPS requests")
-	indexURL       = iniConf.String("indexURL", "https://downloads.arduino.cc/packages/package_staging_index.json", "The address from where to download the index json containing the location of upload tools")
+	indexURL       = iniConf.String("indexURL", "https://downloads.arduino.cc/packages/package_index.json", "The address from where to download the index json containing the location of upload tools")
```

Comment:
Should we change this in this PR?

Reply:
The original work of this PR started because we wanted to address the Private issue related to the default value of the index URL. I agree that neither the PR title nor the commit name makes this change transparent. I'll split this change in a dedicated commit, and Merging with `merge` strategy should make the history clear.